group_vars/lib_jobs/common.yml

```diff
@@ -22,3 +22,7 @@ passenger_ruby: /usr/local/bin/ruby
 ruby_version_default: "ruby-3.1.2"
 ubuntu_ruby_version_default: "ruby-3.1.2"
 desired_nodejs_version: v16.19.0
+# shared rails variables
+rails_app_vars:
+  - name: HONEYBADGER_ENV
```

Comment:
Thanks, @acozine !  After I ran this on staging, the `HONEYBADGER_ENV` var wasn't set.  I wonder if `rails_app_vars` is getting overriden?

Reply:
It looks like lib_jobs is defining the rails_app_vars in `roles/lib_jobs/vars/main.yml`, and I think when you put it in `common.yml`, it doesn't just overwrite one value, it overwrites the whole list.